spec/features/step_by_step_actions_spec.rb

```diff
@@ -4,13 +4,18 @@
   include CommonFeatureSteps
   include NavigationSteps
   include StepNavSteps
+  include Warden::Test::Helpers
 
   before do
     given_I_am_a_GDS_editor
     given_I_can_access_unreleased_features
     setup_publishing_api
   end
 
+  after :each do
+    Warden.test_reset!
+  end
```

Comment:
I've ended up doing this in line 21 of `rails_helper` in my other branch, which then makes the Warden helpers available in any test files (my other branch tests in multiple files IIRC):

```ruby
  config.include Warden::Test::Helpers

  config.after :each do
    Warden.test_reset!
  end
```

Might be cleaner to do that up front here rather than moving it later.